genie-web/src/test/groovy/com/netflix/genie/web/exceptions/checked/AgentLaunchExceptionSpec.groovy

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
At some point we may want to collapse all these boilerplate tests into one like I'm doing for `AgentExecutionExceptionsSpec`